store/src/test/java/org/apache/rocketmq/store/ConsumeQueueTest.java

```diff
@@ -127,7 +150,11 @@ protected void putMsg(DefaultMessageStore master) throws Exception {
         long totalMsgs = 200;
 
         for (long i = 0; i < totalMsgs; i++) {
-            master.putMessage(buildMessage());
+            if (i < totalMsgs/2) {
```

Comment:
Please check the code format even if in the test class.

Reply:
Ok, I have finished the work.